src/index.tsx

```diff
@@ -7,8 +7,9 @@ import NavBar from "./shared/NavBar";
 
 const ChatPage = lazy(() => import("./pages/Chat"));
 const CharacterSettings = lazy(() => import("./pages/CharacterSettings"));
-
-const Home: Component = () => <h1>haven't made a homepage</h1>;
+const Home = lazy(() => import("./pages/Home"));
+const Account = lazy(() => import("./pages/Account"));
+const Register = lazy(() => import("./pages/Account/register"));
```

Comment:
Could we maybe get `register.tsx` renamed to `Register.tsx` and `login.tsx` to `Login.tsx` to match the casing of the other filenames?